### ISSUE TITLE
Sort order for displaying trials not intuitive

### ISSUE THREAD

**Issue by kjcole** [mjsir911/ConfigMaker#33]
When using the presets configurator (and possibly the ratings configurator as well), it displays the presets on the left in an order that does not match the trial order.  In other words, if the preset files are 

    01-restaurant.json
    02-car.json
    03-office.json

the display order might be:

    Business Office
    Noisy Restaurant
    Quiet Car

but should be:

    Noisy Restaurant
    Quiet Car
    Business Office

In other words, the descriptions should appear in trial order. (And if the same problem exists in ratings, the questions should appear in "question order".)
